mindjunkies/courses/views.py

```
@@ -170,7 +170,8 @@ def course_details(request: HttpRequest, slug: str) -> HttpResponse:
     course = get_object_or_404(Course, slug=slug)
     ratings = course.get_individual_ratings()
     enrolled = False
-
+    num_lectures = course.modules.aggregate(models.Count("lectures"))["lectures__count"]
+   
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)